src/ert/config/ert_config.py

```diff
@@ -688,6 +691,8 @@ def handle_default(fm_step: ForwardModelStep, arg: str) -> str:
                         f"forward model step {fm_step.name}: {str(exc)}"
                     ).set_context(fm_step.name)
                 )
+            except ForwardModelStepValidationWarning as exc:
```

Comment:
Instead of being thrown we would like this to be a warning category so that we can

```python
import warnings

warnings.warn(ForwardModelStepWarning("message"))
```

It would then be enough for ForwardModelStepWarning to be a subtype of ConfigWarning:

```
class ForwardModelStepWarning(ConfigWarning)
```

Reply:
Yes, I think thats a good idea